JDI.Light/JDI.Light/Elements/Init/EntitiesCollection.cs

```diff
@@ -0,0 +1,12 @@
+﻿using JDI.Light.Interfaces.Base;
+using JDI.Light.Interfaces.Composite;
+using System.Collections.Generic;
+
+namespace JDI.Light.Elements.Init
+{
+    public class EntitiesCollection
+    {
+        public static Dictionary<string, IPage> Pages = new Dictionary<string, IPage>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5509587)